functions.inc.php

```diff
@@ -1105,7 +1109,7 @@ function _pacrypt_dovecot($pw, $pw_db = '')
 
     $pipes = [];
 
-    $pipe = proc_open("$dovecotpw '-s' $method$dovepasstest", $spec, $pipes);
+    $pipe = proc_open("$dovecotpw '-s' $method$dovepasstest$doveadm_options", $spec, $pipes);
```

Comment:
i think this needs rewriting to be a little clearer - e.g. 

.... {$method}{$dovepasstest}{$doveadm_options} ?

I wonder why -s has quote marks around it?